properties/P000070.md

```diff
@@ -1,8 +1,19 @@
 ---
 uid: P000070
 name: Markov Menger
+aliases:
+  - Markov $\omega$-Menger
+  - Markov $\Omega$-Menger
 refs:
   - doi: 10.14712/1213-7243.2015.201
     name: Applications of limited information strategies in Menger's game
+  - doi: 10.1016/j.topol.2019.02.062
+    name: Relating games of Menger, countable fan tightness, and selective separability
+  - doi: 10.1016/j.topol.2019.07.008
+    name: Limited information strategies and discrete selectivity (Clontz & Holshouser)
 ---
-The second player has a Markov winning strategy in the Menger game (relying on only the round number and most recent move of the opponent). See Game 1.5 and Definition 2.2 of {{doi:10.14712/1213-7243.2015.201}} for more details.
+Markov Menger: The second player has a Markov winning strategy in the Menger game (relying on only the round number and most recent move of the opponent). See Game 1.5 and Definition 2.2 of {{doi:10.14712/1213-7243.2015.201}} for more details.
+
+Markov $\omega$-Menger: The second player has a Markov winning strategy in the game $\mathsf{G}_{\mathrm{fin}}(\Omega_X,\Omega_X)$ (relying on only the round number and most recent move of the opponent). See pages 2 and 3 of {{doi:10.1016/j.topol.2019.07.008}} for more details.
+
+The equivalence of Markov Menger with Markov $\omega$-Menger is established in {{mathse:4730419}}.
```

Comment:
Throughout: add `mathse:4730419` to `refs` when reference

Reply:
Check